paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h

```diff
@@ -180,6 +186,59 @@ class FusedEmbeddingSeqPoolGradKernel : public framework::OpKernel<T> {
   void Compute(const framework::ExecutionContext &context) const override {
     auto *table_var = context.InputVar("W");
     DDim table_dim;
+
+#if defined(PADDLE_WITH_MKLML) && !defined(_WIN32) && !defined(__APPLE__) && \
+    !defined(__OSX__) && !defined(PADDLE_WITH_CUDA)
+    if (table_var->IsType<LoDTensor>()) {
+      table_dim = context.Input<LoDTensor>("W")->dims();
+    } else {
+      PADDLE_THROW(
+          "The parameter W of a FusedEmbeddingSeqPoolGradKernel "
+          "must be either LoDTensor");
+    }
+    auto *ids = context.Input<LoDTensor>("Ids");
+    auto *d_output = context.Input<LoDTensor>(framework::GradVarName("Out"));
+    auto *d_table = context.Output<LoDTensor>(framework::GradVarName("W"));
+    int64_t padding_idx = context.Attr<int64_t>("padding_idx");
+
+    d_table->Resize(table_dim);
+    auto *d_table_data = d_table->mutable_data<T>(context.GetPlace());
+    memset(d_table_data, 0, d_table->numel() * sizeof(T));
```

Comment:
d_table_data will be the output of sparse GEMM. Do we really need memset it in advance?

Reply:
done